website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+
+### Highlights
+
+- [Metro support for Package Exports, symlinks, and config changes](/blog/2023/05/30/0.72-metro-package-exports-symlinks#metro)
+- [Developer experience improvements](/blog/2023/05/30/0.72-metro-package-exports-symlinks#developer-experience-improvements)
+- [Moving New Architecture updates](/blog/2023/05/30/0.72-metro-package-exports-symlinks#react-native-package-renames)
+- [Mitigation for TextInput ANRs on Android](/blog/2023/05/30/0.72-metro-package-exports-symlinks#react-native-package-renames)
+
+### Breaking Changes
+
+- [Deprecated Component Removals](/blog/2023/05/30/0.72-metro-package-exports-symlinks#deprecated-component-removals)
+- [Package renames](/blog/2023/05/30/0.72-metro-package-exports-symlinks#package-renames)
+
+<!--truncate-->
+
+## Highlights
+
+### Metro
+
+#### Symlink support (Beta)
+
+Symlink support continues to be one of the top-requested features in Metro and for React Native 0.72, Metro introduces pre-stable support for resolving symlinks.
+
+Symlink support enables React Native to work transparently with monorepo setups and pnpm, removing the need for workarounds. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+#### Package Exports support (Beta)
+
+[Package Exports](https://nodejs.org/api/packages.html#exports) is the modern alternative to the package.json "main" field and provides new capabilities for npm packages to define their public API and target React Native.
+
+By enabling Package Exports support in your Metro config, your app will be compatible with the wider JavaScript ecosystem, including via the new "react-native" community condition. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+:::tip
+See the newly released [blog post](https://reactnative.dev/blog/2023/05/08/package-exports-support) to learn more about this feature and our plans for stable rollout.
+:::
+
+#### New Metro config file setup
+
+In React Native 0.72, we’ve changed the config loading setup for Metro in React Native CLI. Please update your project’s `metro.config.js` file to match the [template’s version](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js).
+
+:::info
+Please update your config file to following [format](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js). Changes to `metro.config.js` will become required in 0.73. For 0.72, we will log a warning if not updated.
```

Comment:
```suggestion
Please update your config file to following [format](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js). These format changes to `metro.config.js` will become required in 0.73. For 0.72, we will log a warning if not updated. You can also follow the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/?from=0.71.8&to=0.72.0).
```
(headsup, that upgrade-helper url will only work proper once 0.72.0 is out, for now it will redirect to main UH page